### DISCUSSION TITLE
Is fetch a global implementation or unique to each route?

### DISCUSSION THREAD

**Discussion by NullzInc** [sveltejs/kit#8285] [Q&A]
I'm using urql and urql can accept a fetch implementation when creating a urql client. SK complains when urql uses `window.fetch`. This got me thinking...

I'm wondering if it would be acceptable to create a urql client with a copy fetch from a hook or load function, or is the fetch returned from load function unique to the load function in some way? The docs talk about tracking data, which makes me wonder if the fetch for each load function is somehow unique?

Thanks!

**Comment by wvhulle** [comment]
Have you got an answer? I would like to use fetch outside load functions.
